rollup.config.js

```diff
@@ -0,0 +1,115 @@
+import alias from 'rollup-plugin-alias'
+import babel from 'rollup-plugin-babel'
+import commonjs from 'rollup-plugin-commonjs'
+import resolve from 'rollup-plugin-node-resolve'
+import peerDepsExternal from 'rollup-plugin-peer-deps-external'
+import replace from 'rollup-plugin-replace'
+import { terser } from 'rollup-plugin-terser'
+
+const path = require('path')
+const fs = require('fs')
+const { join } = require('path')
+const { readdirSync } = require('fs-extra')
+
+const NODE_ENV = process.env.NODE_ENV || 'development'
+const extensions = ['.web.js', '.js']
+
+function removeExt(path) {
+  return path.replace(/\.[^.]+$/, '')
+}
+
+const res = readdirSync('src/components').reduce((acc, filename) => {
+  const path = join('src/components/', filename)
+  let file = ''
+  if (fs.existsSync(join(path, 'index.web.js'))) {
+    file = join(path, 'index.web.js')
+  } else {
+    file = join(path, 'index.js')
+  }
+  return {
+    ...{ [removeExt(filename)]: file },
+    ...acc,
+  }
+}, {})
+
+const getPlugins = (minify = true) => [
+  replace({
+    'process.env.NODE_ENV': JSON.stringify(NODE_ENV),
+  }),
+  babel({
+    exclude: 'node_modules/**',
+  }),
+  alias({
+    resolve: ['.web.js', '.js', '/index.js'],
+    components: path.join(__dirname, 'src/components'),
+    helpers: path.join(__dirname, 'src/helpers'),
+  }),
+  resolve({ extensions, preferBuiltins: false }),
+  commonjs(),
+  peerDepsExternal(),
+  process.env.NODE_ENV === 'production' && minify ? terser() : '',
+]
+
+const getGlobals = () => ({
```

Comment:
No need for a function if you don't have arguments, you can use a const I reckon

Reply:
👍 Done